Code/Framework/AzToolsFramework/Tests/ManipulatorBoundsTests.cpp

```diff
@@ -187,4 +187,51 @@ namespace UnitTest
         EXPECT_NEAR(intersectionDistance, 10.0f, g_epsilon);
         EXPECT_TRUE(intersection);
     }
+
+    // replicates a scenario in the Editor using a cone and a pick ray which should have failed but passed with Intersect::IntersectRayCone
+    TEST(ManipulatorIntersectRayConeTest, RayConeEditorScenarioTest)
+    {
+        auto rayOrigin = AZ::Vector3(0.0f, -0.808944702f, 0.0f);
+        auto rayDir = AZ::Vector3(0.301363617f, 0.939044654f, 0.165454566f);
+        float t = 0.0f;
+        bool hit = AzToolsFramework::Picking::IntersectRayCone(
+            rayOrigin, rayDir, AZ::Vector3(0.0f, 0.0f, 0.161788940f), AZ::Vector3(0.0f, 0.0f, -1.0f), 0.0453009047, 0.0113252262, t);
+        EXPECT_EQ(hit, false);
+    }
+
+    // cone lying flat, ray going towards base of cone
+    TEST(ManipulatorIntersectRayConeTest, RayIntersectsConeBase)
+    {
+        auto rayOrigin = AZ::Vector3::CreateZero();
+        auto rayDir = AZ::Vector3::CreateAxisY();
+        float t = 0.0f;
+        bool hit = AzToolsFramework::Picking::IntersectRayCone(
+            rayOrigin, rayDir, AZ::Vector3::CreateAxisY(10.0f), AZ::Vector3::CreateAxisY(-1.0f), 5.0f, 1.0f, t);
+        EXPECT_EQ(hit, true);
+        EXPECT_THAT(t, ::testing::FloatNear(5.0f, 0.0001f));
+    }
+
+    // cone standing up, ray going towards mid side of cone
+    TEST(ManipulatorIntersectRayConeTest, RayIntersectsConeSide)
```

Comment:
Suggestion: Would also be nice to have a disabled test showing the algorithm-failing case (ray starts inside code).

Reply:
I'm actually against this as this is not a bug, it's just something the code hasn't been updated to do yet, so adding that here I don't think is valid, but it could be added in future if we wish to expand the functionality